src/components/login/SignupBox/SignupBox.jsx

```diff
@@ -41,14 +113,17 @@ const SignupBox = () => {
             id="signup-btn"
             type="button"
             className="btn-submit btn-solid-primary"
+            onClick={() => {
+              doSignup({ email, password });
+            }}
           >
             Create new account
           </button>
         </div>
         <Link to="/login">
-        <label className="form-input-label have-acc">
-          Already have an account?
-        </label>
+          <label className="form-input-label have-acc">
+            Already have an account?
+          </label>
         </Link>
       </form>
```

Comment:
Suggestion: You can include name information for signup purpose.

Reply:
Ok
I will include this in future release as the API for now also takes only email and password